sdks/python/test-suites/tox/py37/build.gradle

```diff
@@ -35,6 +35,9 @@ lint.dependsOn lintPy37
 toxTask "mypyPy37", "py37-mypy"
 lint.dependsOn mypyPy37
 
+toxTask "formatter", "py3-yapf-check"
+check.dependsOn formatter
```

Comment:
I missed this in my review. The lint precommit runs :pythonLintPreCommit, which depends on `lint` tasks.
Suggestion:
```
lint.dependsOn formatter
```

Reply:
Nevermind, I see there's a new job named PythonFormatter